packages/lit-dev-content/site/docs/ssr/overview.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Server-side rendering (SSR)
+eleventyNavigation:
+  key: Overview
+  parent: Server rendering
+  order: 1
+---
+
+{% labs-disclaimer %}
+
+Server-side rendering (SSR) is a technique for generating and serving the HTML of your components, including shadow DOM and styles, before their JavaScript implementations have loaded and executed.
+
+You can use SSR for a variety of reasons:
+- Performance. Some sites can render faster if they render static HTML first without waiting for JavaScript to load, then (optionally) load the page's JavaScript and hydrate the components.
+- SEO and web crawlers. While the major search-engine web crawlers render pages with full JavaScript-enabled browsers, not all web crawlers support JavaScript.
+- Robustness. Static HTML renders even if the JavaScript fails to load or the user has JavaScript disabled.
+
+For a deeper dive into server-side rendering concepts and techniques generally, see [Rendering on the Web](https://web.dev/rendering-on-the-web/) on web.dev.
+
+Lit supports server-side rendering through the [Lit SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#readme) package. Lit SSR renders Lit components and templates to static HTML markup in non-browser JavaScript environments like Node. It works without fully emulating the browser's DOM, and takes advantage of Lit's declarative template format to enable fast performance, low time-to-first-byte, and support streaming.
```

Comment:
I would add a section here on status of the library that includes a disclaimer about this library being under very active development and listing some current, hopefully temporary limitations:

- Async component work isn't supported (link to issue)
- isSSR not being available yet (link to issue)
- Declarative shadow DOM (and move most detail to "Client Usage"
- Needing support for the "node" export condition
- TBD Integrations
- ElementRendererRegistry needing to be figured out

Reply:
I'm not sure if "node" export condition is as much of a limitation worth mentioning here. It's not something that's fixable and it's been available in Node unflagged since 12.16.0, 13.7.0. Major build tools all seem to recognize it and have options for it too.

I've added the rest with links to relevant issues or other parts of the doc.